src/components/SkipLink/SkipLink.ts

```diff
@@ -0,0 +1,71 @@
+export default class SkipLink extends HTMLElement {
+    private $link: HTMLLinkElement | null;
+
+    constructor() {
+        super();
+
+        // Binding
+        this.onClick = this.onClick.bind(this);
```

Comment:
Rebinding is unnecessary if you declare your method as an arrow function:

```diff
- onClick(e: Event) {
+ onClick: (e: Event) => {
```

Reply:
To my understanding that syntax is not supported inside of a `class`. Unless i'm missing something? 

This would work if we moved the function inside the constructor but I'm not found of the idea

Reply:
I'm not fond of moving it to the constructor either.

It does work within classes. I've used it in personal projects.

Reply:
Oh 🤦 Yes. `=` and not `:`.